### ISSUE TITLE
Error when opening AppImage

### ISSUE THREAD

**Issue by namelessperson0** [radicle-dev/radicle-upstream#2731]
Getting below error when launching AppImage
```
{
  "stack": "Error: Failed to load the session\n    at vo (file:///tmp/.mount_radiclCKJX2t/resources/app.asar/public/bundle.js:1:1692448)\n    at async _o (file:///tmp/.mount_radiclCKJX2t/resources/app.asar/public/bundle.js:1:1692561)",
  "message": "Failed to load the session",
  "code": "SessionFetchFailure",
  "source": {
    "stack": "TypeError: Failed to fetch\n    at Zt.fetch (file:///tmp/.mount_radiclCKJX2t/resources/app.asar/public/bundle.js:1:1650401)\n    at Zt.fetchOk (file:///tmp/.mount_radiclCKJX2t/resources/app.asar/public/bundle.js:1:1649923)\n    at Object.sessionGet (file:///tmp/.mount_radiclCKJX2t/resources/app.asar/public/bundle.js:1:1650935)\n    at t (file:///tmp/.mount_radiclCKJX2t/resources/app.asar/public/bundle.js:1:1692088)\n    at d (file:///tmp/.mount_radiclCKJX2t/resources/app.asar/public/bundle.js:1:1591826)\n    at async vo (file:///tmp/.mount_radiclCKJX2t/resources/app.asar/public/bundle.js:1:1692072)\n    at async _o (file:///tmp/.mount_radiclCKJX2t/resources/app.asar/public/bundle.js:1:1692561)",
    "message": "Failed to fetch",
    "code": "UnknownException",
    "details": {
      "name": "TypeError"
    }
  }
}
```

Attaching screenshot below
![radicleError](https://user-images.githubusercontent.com/72940077/150136160-9816dfa8-e71c-4710-bdcc-a8477b1cead4.png)

**Comment by geigerzaehler** [comment]
Hi @namelessperson0 :wave:. Upstream fails to run because your system uses an older version of glibc than we compiled it for. Would you mind sharing what OS your running and what version of glibc you have installed?

**Comment by geigerzaehler** [comment]
I’ve also created an issue to improve this #2732

**Comment by namelessperson0** [comment]
Debain Buster 10.11 with GLIBC 2.28-10

**Comment by namelessperson0** [comment]
> Hi @namelessperson0 wave. Upstream fails to run because your system uses an older version of glibc than we compiled it for. Would you mind sharing what OS your running and what version of glibc you have installed?

Debain Buster 10.11 with GLIBC 2.28-10

**Comment by geigerzaehler** [comment]
I’ve built the AppImage with an earlier version of Ubuntu (#2737). You can find it [here](https://storage.googleapis.com/radicle-upstream-build-artifacts/v1/by-commit/292bf4be3078806a4772585f548c53bd4635c5ed/radicle-upstream-0.2.13.AppImage). Could you try it out and see if it works for you @namelessperson0?

**Comment by geigerzaehler** [comment]
This should be closed with #2737. Please reopen if it is not fixed.
